Mathlib/Algebra/Ring/Int.lean

```diff
@@ -134,6 +134,14 @@ lemma not_even_two_mul_add_one (n : ℤ) : ¬ Even (2 * n + 1) :=
 lemma even_sub' : Even (m - n) ↔ (Odd m ↔ Odd n) := by
   rw [even_sub, ← not_odd_iff_even, ← not_odd_iff_even, not_iff_not]
 
+@[simp]
+lemma odd_add_one {n : ℤ} : Odd (n + 1) ↔ ¬ Odd n := by
```

Comment:
IMHO, `Even n` is a more natural RHS, and it can be shown for any add monoid with cancellable addition.

Reply:
Note that we have `Nat.even_add_one` and `Int.even_add_one` that say `Even (n + 1) ↔ ¬Even n`.